### PR TITLE
Phase2-hgx359H Step to include the new HD scintillator in the V19 setup of HGCal

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Step to include the new HD scintillator in the V19 setup of HGCal

#### PR validation:

All earlier versions are tested with runTheMatrix

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/src/HGCalCassette.cc [hunk 1]
```diff
@@ -3,20 +3,29 @@
 #include <algorithm>
 #include <sstream>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline Geometry/HGCalCommonData/src/HGCalCassette.cc:6]: @bsunanda , please, comment it out.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@civanch @srimanob Please approve this PR

**Comment by srimanob** [comment]
+Upgrade

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
+geometry

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**